lib/TypeNames/ModelToPTMLTypeHelpers.cpp

```diff
@@ -290,10 +258,8 @@ void printForwardDeclaration(const model::TypeDefinition &T,
 static void printDefinition(const model::EnumDefinition &E,
                             ptml::PTMLIndentedOstream &Header,
                             ptml::PTMLCBuilder &B,
-                            const TypeSet &TypesToInline,
-                            llvm::StringRef NameOfInlineInstance,
-                            const std::vector<model::Qualifier> &Qualifiers,
-                            bool ForEditing) {
+                            bool ForEditing,
+                            std::string &&Suffix = {}) {
```

Comment:
Be consistent. You're defaulting `Suffix` to `{}` here and to `""` below. Pick one style and use it everywhere.

Reply:
Done.